libs/blockchain/src/blockchain.service.ts

```diff
@@ -181,10 +181,10 @@ export class BlockchainService extends BlockchainRpcQueryService implements OnAp
     return siwfTxnValues;
   }
 
-  public async validateProviderSeedPhrase() {
-    const { providerSeedPhrase, providerId } = this.config;
-    if (providerSeedPhrase) {
-      const address = await addressFromSeedPhrase(providerSeedPhrase);
+  public async validateproviderKeyUriOrPrivateKey() {
+    const { providerKeyUriOrPrivateKey, providerId } = this.config;
+    if (providerKeyUriOrPrivateKey) {
+      const address = await addressFromSeedPhrase(providerKeyUriOrPrivateKey);
```

Comment:
This doesn't seems to support ethereum key path

Reply:
This is just on the provider side. We don't need that yet, but it should be addressed soon. @shannonwells want to throw in an issue for supporting eth keys for providers?